includes/class-base-widget.php

```diff
@@ -397,9 +397,98 @@ protected function render_form_textarea( array $field ) {
 	}
 
 	/**
+	 * Render the 'Day' div and select fields
+	 *
+	 * @param  array $field Field data
+	 * @param  array $day   The current day in the iteration.
+	 * @param  array $hours The array of times.
+	 *
+	 * @since NEXT
+	 *
+	 * @return mixed
+	 */
+	protected function render_day_input( $field, $day, array $hours ) {
+
+		$field['name'] = str_replace( 'value', strtolower( $day ), $field['name'] );
```

Comment:
@EvanHerman I think you can align the equal sign of those 2 assignation since they deal with fields elements.